_events/repeated/dei-working-group-call.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Diversity, Equity, and Inclusion Working Group Call
+subtitle:
+location: Virtual
+expires: 2022-07-31
+event_date: "every other Thursday at 15:00-16:00 Eastern"
+layout: event
+category: dei
+time:
+  - - start: 2022-01-13 15:00 EST
+
+repeated: true
+rrule:
+  - DTSTART;TZID=America/New_York:20220113T150000
+  - RRULE:FREQ=WEEKLY;INTERVAL=2;WKST=MO;BYDAY=TH
+  - RDATE;TZID=America/New_York:20220113T150000
+---
+
+The Diversity, Equity, and Inclusion Working Group meetings occurs bi-weekly on
+Thursdays each month at 15:00-16:00 Eastern. The Google Drive Folder for
+agendas and notes is available
+[here](https://drive.google.com/drive/u/1/folders/1apoD9cbYeINxf3J9CNrznwAlAPVAkGBm?ths=true).
+Please get the Zoom link by asking for it in the dei-discussion channel on
+slack.
```

Comment:
```suggestion
The Diversity, Equity, and Inclusion Working Group meets every other Thursday.

See the [DEI Working Group webpage](https://us-rse.org/wg/dei/) for more information about the group and how to get involved.

You can also find us in , visit the [`#dei-discussion` channel](https://usrse.slack.com/messages/dei-discussion) on the US-RSE Slack workspace.
```

Reply:
Thanks @lparsons. I will work on adding these changes. Since the date isn't quite right with our current system (see below) do you think line 19 above should still contain the time?

In terms of the repeated vs repeating. I think we are going to address this in a future issue/PR. It seems our repeating events system needs some adjustments to work with the new RRULE setup we have to make the calendar work. As you can see, this is only using our initial date (the first event of the reoccuring events) and not the nearest upcoming date. Nor is it really providing the schedule, other than in the description. I will make sure to add an issue to address these, but in the meantime at least change the phrasing to repeating everywhere.

Reply:
> Thanks @lparsons. I will work on adding these changes. Since the date isn't quite right with our current system (see below) do you think line 19 above should still contain the time?

I didn't realize that there were issues with the date/time on repeating events. In that case, it makes sense to include the time. *e.g.*

```markdown
The Diversity, Equity, and Inclusion Working Group meets for one hour, every other Thursday at 3 pm (ET), 6 pm (PT).
```

> In terms of the repeated vs repeating. I think we are going to address this in a future issue/PR. ... I will make sure to add an issue to address these, but in the meantime at least change the phrasing to repeating everywhere.

Sounds good to me!

Reply:
What is the issue?

Reply:
Sorry about the delay. Been having some computer issues and had to prioritize that. The issue I was specifically mentioning here was the fact that we use the
```
time:
  - - start: sometime
  - - end: another time
```
variables for all events to display the date/time of the event on the given event page. Since these variables have no knowledge of the repeating nature of the event, it will always display the first date this takes place. You can see in this PR that the DEI-WG event says the event is on Thursday, January 13, 2022, 3:00PM (see the screenshot Lance attached). Obviously this date has passed. I figured ideally we'd like to have this date/time be the occurrence of the next upcoming event, rather than the "first in the series", so to speak. So I was hoping we'd be able to figure out how to propagate this date/time via the RRULE we have in place for repeated events.

Reply:
We only loop through that list given that the RRULE isn't set, so if the time isn't set I don't think we will render anything. That loop could definitely be refactored, and figure out how to get the next date to properly render. It's also based on when the site is rendered, so if it's not rendered for a long time (even if we get it to work) it might not be the correct next date.